### DISCUSSION TITLE
Motr REST API server and a golang http client

### DISCUSSION THREAD

**Discussion by kiwionly2** [Seagate/cortx-motr#2066] [Show and tell]
Hi all, 

I had wrote a REST API server for Motr with a golang http client. The api server is based on go bindings for motr. 

You could check out the code at this [branch](https://github.com/kiwionly2/cortx-motr/tree/go_api_server/bindings/go/go_motr_api_server) and the readme for more detail.

Cheers !

**Comment by hessio** [comment]
Nice @kiwionly2 - can you please try to merge this to main and let's see if we can get this added to the cortx-motr repo

**Reply by kiwionly2** [reply]
Sure @hessio, I have create a PR for this : #2111

**Reply by hessio** [reply]
Thanks looks good!
